pkg/gofr/gofr_test.go

```diff
@@ -562,6 +562,20 @@ func Test_AddCronJob_Success(t *testing.T) {
 	assert.Truef(t, pass, "unable to add cron job to cron table")
 }
 
+func Test_Shutdown(t *testing.T) {
+	g := New()
+
+	g.GET("/hello", func(*Context) (interface{}, error) {
+		return helloWorld, nil
+	})
+
+	go g.Run()
+	time.Sleep(2 * time.Second)
```

Comment:
this delay is huge for tests

100 ms would be enough

Reply:
There are a lot of leftovers like this. Probably could be done better after implementing readiness probe